ddl/ddl_db_test.go

```diff
@@ -1118,3 +1118,48 @@ func (s *testDBSuite) TestIssue2858And2717(c *C) {
 	s.tk.MustQuery("select a from t_issue_2858_hex").Check(testkit.Rows("291", "123", "801"))
 	s.tk.MustExec(`alter table t_issue_2858_hex alter column a set default 0x321`)
 }
+
+func (s *testDBSuite) TestChangeColumnPosition(c *C) {
```

Comment:
You can add a test case that check `show create table t;` or `desc t;` result after modify column position.